model.go

```diff
@@ -55,3 +55,9 @@ type YaraRule struct {
 	Description *string `json:"description"`
 	Reference   *string `json:"reference"`
 }
+
+type upload struct {
```

Comment:
Comment?
Seems like this is the upload API schema for abuse.ch

Reply:
fixed